ding/envs/env_manager/env_supervisor.py

```diff
@@ -255,6 +260,8 @@ def ready_obs(self) -> tnp.array:
             >>> timesteps = env_manager.step(action)
         """
         active_env = [i for i, s in self._env_states.items() if s == EnvState.RUN]
+        if self._return_original_data:
```

Comment:
why add this

Reply:
In the comment, I explained it as "return original observation,  so that the attribute self._ready_obs is not a tnp.array but only the original observation, and the property self.ready_obs is a dict in which the key is the env_id." In fact the reason why I add this attribute is:
1. Sometimes for multi-envs we cannot get observation for every envs, so a dict instead of list is necessary, to directly get the env_id of obs. **PS**: And it is strange that,  for the return timesteps of step(action), the return of EnvSupervisor and BaseEnvManagerV2 is list, and the returns subprocesEnvManager and BaseEnvManager is dict. I think we need to unify them to dict.
2. For envs like DI-star, the tnp.array will raise bug in EnvSupervisor, I forgot which specific bug it is, but it made me to return directly obs instead of tnp.array.  I will try to reproduce this bug if necessary